src/lang/proc_macros/plugins/scarb/inline.rs

```diff
@@ -20,17 +21,21 @@ pub fn inline_macro_generate_code(
     let stable_ptr = syntax.clone().stable_ptr().untyped();
     let arguments = syntax.arguments(db);
     let token_stream = TokenStream::from_syntax_node(db, &arguments);
-
     // region: Modified scarb code
-    let result = db.get_inline_macros_expansion(ExpandInlineMacroParams {
-        context: expansion_context,
-        name: syntax.path(db).as_syntax_node().get_text_without_trivia(db),
-        args: token_stream,
-    });
+    let result = get_inline_macros_expansion(
```

Comment:
Call on `db` so salsa can kick in
```suggestion
    let result = db.get_inline_macros_expansion(
```